iroh-bitswap/src/lib.rs

```diff
@@ -287,6 +288,7 @@ impl<S: Store> Bitswap<S> {
 
     fn receive_message(&self, peer: PeerId, message: BitswapMessage) {
         inc!(BitswapMetrics::MessagesReceived);
+        record!(BitswapMetrics::MessageBytesIn, message.encoded_len() as u64);
```

Comment:
this encodes the message again :/

Reply:
Ik :( Want me to extend BitswapMessage with a pre calculated len?

Reply:
maybe we could cache it?